packages/component-library/draft/Kaizen/Popover/Popover.elm

```diff
@@ -170,54 +175,136 @@ view (Config config) =
                 Nothing ->
                     text ""
 
-        ( sideClass, positionClass ) =
-            mapPositionToClass config.tipPosition
+        ( side, position ) =
+            config.tipPosition
     in
     div
-        [ styles.classList
-            [ ( mapVariantToRootClass config.variant, True )
-            , ( sideClass, True )
-            , ( positionClass, True )
-            ]
+        [ styles.class .root
+        , getRootStyle
         ]
-        [ case config.heading of
-            Just heading ->
-                div [ styles.class .header ]
-                    [ span [ styles.class .icon ] [ mapVariantToIcon config.variant ]
-                    , text heading
-                    , viewClose
-                    ]
-
-            Nothing ->
-                text ""
-        , div [ styles.class .container ] [ config.content ]
+        [ div
+            [ styles.classList
+                [ ( mapVariantToBoxClass config.variant, True ) ]
+            ]
+            [ case config.heading of
+                Just heading ->
+                    div [ styles.class .header ]
+                        [ if config.variant == Default then
+                            span [] []
```

Comment:
I wasn't sure how to return no element here

Reply:
If it's returning an `Html Msg`, you can do `text ""` to return a blank text node if you want to avoid random empty stuff in the DOM.

Reply:
```suggestion
                            text ""
```